src/config.rs

```diff
@@ -1383,6 +1384,14 @@ impl Config {
         Ok(config)
     }
 
+    /// Override configurations based on CLI options.
+    pub fn flag(mut self, options: &Options) -> Self {
+        if options.title.is_some() {
+            self.dynamic_title = false;
+        }
+        self
+    }
```

Comment:
I think you might have misunderstood me? I don't think we need a function to override configs with CLI options if all we want to do is update the dynamic title. A function that more accurately describes what's going on should be appropriate.

I was more thinking about something like this:
```rust
pub fn disable_dynamic_title(&mut self) {
    self.dynamic_title = false;
}
```

This is a super specific method that accurately displays what's going on without having to move the object or provide any unnecessary mutability.

Reply:
I didn't misunderstand, I chose to do it like this because it's a good solution. The `flag` function name might need to be changed (I like it) but it gives a single place to override configurations from flags, something I'm sure will come up again. Instead of needing a handful of setter functions individually.

Reply:
If we would want a way to override more configuration flags from options, a more generic way to do so would definitely be our bet. I don't think there's any other command line parameter currently in alacritty that would need to override any config option.

Currently this just looks like additional overhead to me. It's used to do just a simple thing trying to help out in the future, but I don't see this being super relevant in the future, so it's just added complexity.

If we would still want to do it that way, the `flag` name is very non-descriptive though, there's no way to tell from the function signature what it actually does, so I'd say it definitely has to be renamed at least.